guides/v2.1/cloud/cdn/configure-fastly.md

```diff
@@ -35,43 +35,52 @@ The process for configuring Fastly includes:
 
-*	Fastly Service ID
-*	Fastly API token
+-  For Starter projects, include the origin URLs for your Master and Staging sites.
 
-You can also locate these credentials in your Staging and Production systems
-in `/mnt/shared/fastly_tokens.txt`. You can SSH into the servers to verify the
-file in that location. If you do not locate this file, please enter a ticket
-for [Support]({{ page.baseurl }}/cloud/trouble/trouble.html) asking to have the
-file added. We can help provide this credentials file.
+If the domains included in your Cloud project are associated with an existing Fastly service account, see [Multiple Fastly accounts and assigned domains](#domain).
 
-{:.bs-callout .bs-callout-warning}
-Make note of which environment each set of credentials is used for. If you use
-the wrong credentials in an environment, you'll encounter issues with Fastly.
+
+####  To view your Fastly credentials:
+
+When we provision your project environments for Fastly services, we add the Fastly credentials to your project environment configuration. You can get these credentials using any of the following methods:
+
+-  After you [enable the Fastly module](#cloud-fastly-config), you can view the credentials in the Fastly configuration section of the Admin UI.
+
+   ![Fastly credentials Admin UI]({{ site.baseurl }}/common/images/cloud/cloud-fastly-credentials-admin-ui.png){:width="650px"}
+
+-  You can get credentials from the following environment variables that are visible in the [Environment configuration]({{ page.baseurl }}/cloud/project/projects.html#environment-configuration-variables) section of the Project Web UI.
+
+   -  `CONFIG__DEFAULT__SYSTEM__FULL_PAGE_CACHE__FASTLY__FASTLY_API_KEY`
+   
+   -  `CONFIG__DEFAULT__SYSTEM__FULL_PAGE_CACHE__FASTLY__FASTLY_SERVICE_ID`
+
+-  Use the Magento CLI `magento-cloud variable:get -e <environment ID>` command. See [List and review variables]({{ page.baseurl }}/cloud/before/before-setup-env-2_clone.html#variablelist). 
+
+-  On Pro projects, you can find the Fastly credentials in the `/mnt/shared/fastly_tokens.txt` file. You can SSH into the servers to verify the file in that location.
+
+{:.bs-callout .bs-callout-info}
+If you need to change the Fastly API key credential for a project environment for security reasons, you can request a new API key from support.  See [Change Fastly credentials](https://support.magento.com/hc/en-us/articles/360006935271-Change-account-owner-access-credentials-via-API-tokens-for-Fastly-on-Cloud).
+
+### Multiple Fastly accounts and assigned domains {#domain}
+
+If you have an existing Fastly account that has been configured with the same apex and sub-domains used for your {{ site.data.var.ece }} project, remove those domains from your Fastly account before submitting the request to add Fastly services to your project environments. See [Working with Domains](https://docs.fastly.com/guides/basic-configuration/working-with-domains) in the Fastly documentation. 
+
+**DNS delegation**
```

Comment:
I'd like some clarification on this section. I'll meet with you to discuss, @meker12